install_config/install/rpm_vs_containerized.adoc

```diff
@@ -13,24 +13,56 @@ toc::[]
 
 == Overview
 
-This section explores some of the preparation required to install {product-title}
-as a set of services within containers. This applies to hosts using either Red
-Hat Enterprise Linux or Red Hat Atomic Host.
+An RPM installation installs all services through package management and
+configures services to run within the same user space, while a containerized
+installation installs services using container images and runs separate services
+in individual containers.
 
-- For the xref:../../install_config/install/quick_install.adoc#install-config-install-quick-install[quick installation
-method], you can choose between the RPM or containerized method on a per host
-basis during the interactive installation, or set the values manually in an
+The default method for installing {product-title} on
+ifdef::openshift-origin[]
+Fedora, CentOS, or RHEL
```

Comment:
Why mentioned distributions we don't support? This should just say RHEL. Upstream it can read this but for the OCP docs this needs to only say RHEL.